tests/ert3/console/integration/test_cli.py

```diff
@@ -52,6 +52,29 @@ def test_cli_init(tmpdir):
         ert3.console.main()
 
 
+def test_cli_init_example(tmpdir):
+    workspace = tmpdir / _POLY_WORKSPACE_NAME
+    shutil.copytree(_POLY_WORKSPACE, workspace)
+    workspace.chdir()
+
+    args = ["ert3", "init", "--example", "something"]
+    with patch.object(sys, "argv", args):
+        with pytest.raises(SystemExit) as error:
```

Comment:
Should this not be an `IllegalWorkspaceOperation`?

Reply:
I don't think it would work cause in main we catch `sys.exit(e)` or am I wrong and it doesn't matter?

```
def main():
    try:
        _main()
    except ert3.exceptions.ErtError as e:
        sys.exit(e)
```

Reply:
You are correct 👍